java/arcs/core/storage/handle/CollectionImpl.kt

```diff
@@ -69,11 +69,16 @@ class CollectionImpl<T : Referencable>(
 
     /** Returns the values in the collection that fit the requirement (as a set). */
     suspend fun query(args: Any): Set<T> {
-        // Note: type checking for args is implemented in the refinement and code generated handle.
-        requireNotNull(refinement) {
-            "Invalid operation 'query' on collection $name which has no associated query"
+        val results = value().filter { it ->
```

Comment:
If you're sticking with `it`, you can drop the `it ->` from the lambda.

Reply:
Ah, I knew there was a reason I kept seeing `it` lambdas. Thanks